collections/nemo_nlp/nemo_nlp/data/datasets/token_classification.py

```diff
@@ -144,22 +140,22 @@ def get_features(queries,
 
         all_segment_ids.append([0] * max_seq_length)
 
-    logger.info(f'{too_long_count} are longer than {max_seq_length}')
+    nemo.logging.info(f'{too_long_count} are longer than {max_seq_length}')
 
     for i in range(min(len(all_input_ids), 5)):
-        logger.info("*** Example ***")
-        logger.info("i: %s" % (i))
-        logger.info(
+        nemo.logging.info("*** Example ***")
```

Comment:
and that looks for me rather like a debug information...?

Reply:
done